instaclone/src/components/SearchBar.js

```diff
@@ -0,0 +1,39 @@
+import React from "react"
+import "./head.css"
+
+const SearchBar = (props)=>{
+    const style={
```

Comment:
Again, we want to prevent using style variables in this way because it's not immediately apparent to another developer what is meant to be occuring in this component, and it makes your code look much less clean.